openedx/core/djangoapps/content/search/api.py

```diff
@@ -323,6 +323,7 @@ def rebuild_index(status_cb: Callable[[str], None] | None = None) -> None:
             Fields.type,
             Fields.access_id,
             Fields.last_published,
+            Fields.content + ".problem_types",
```

Comment:
It's a bit weird to have this string dangling here.. Is it worth defining `problem_types` as a sub-key of `Fields.content` in `search.documents` like we did [with `tags_levelX`](https://github.com/open-craft/edx-platform/blob/chris/FAL-3781-update-index-problem-types/openedx/core/djangoapps/content/search/documents.py#L48-L51)?

Reply:
Updated here: https://github.com/openedx/edx-platform/pull/35242/commits/4dd2b95fbe3ba2a11d166fb2e02438de3944d58b